### PR TITLE
Added "How to create an event subscriber"

### PR DESCRIPTION
**Pull request by beni0888**
I've added the information about how to create an event subscriber to the "How to create an Event listener" cookbook recipe. I think that there isn't a recipe in the cookbook about that and it can be useful.

### CODE REVIEW

**Comment by WouterJ** [comment]
ping @beni0888 Can you please update this PR? (If you have no time to do it, feel free to see that too, we'll then finish this PR)

**Comment by beni0888** [comment]
@WouterJ yes, I'm sorry I've been quite bussy... I'll try to fix it today :)

File: cookbook/service_container/event_listener.rst [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
+=============================================
 
 Symfony has various events and hooks that can be used to trigger custom
 behavior in your application. Those events are thrown by the HttpKernel
 component and can be viewed in the :class:`Symfony\\Component\\HttpKernel\\KernelEvents` class.
 
 To hook into an event and add your own custom logic, you have to create
-a service that will act as an event listener on that event. In this entry,
-you will create a service that will act as an Exception Listener, allowing
+a service that will act as an event listener on that event. You can do that in two different ways,
```
Comment (cordoval) [inline cookbook/service_container/event_listener.rst]: service that will listen to that event.

**Comment by beni0888** [comment]
@WouterJ I'm very sorry for not to fix the problems in this PR as I promised, but I've been really busy and I haven't been able to do it. However, I keep this task in my mind and I will do it as soon as possible. Sorry.

**Comment by javiereguiluz** [comment]
@beni0888 there is no need to apologize at all! This is a volunteer work and we understand that sometimes we face large workloads which prevents us from dedicating time to open source projects.

Feel free to work on this when you can and, if you think you cannot finish it, please tell us and we'll overtake this pull request. Thanks.

**Comment by beni0888** [comment]
@javiereguiluz :+1:

**Comment by WouterJ** [comment]
@beni0888 this saturday, we'll be hosting a [doc sprint day](http://symfony.com/blog/symfony-docs-hack-day-needs-you-on-may-23rd). Would you like to finish this PR yourself (maybe during the day?) or should we put it on a list so others can finish it?

**Comment by beni0888** [comment]
@WouterJ I think I won't be able to work on this PR this saturday and It's opened for too long, so it would be nice if you put it on the list :)

**Comment by javiereguiluz** [comment]
I've finished this PR in a new one: #5377. I cherry-picked @beni0888's commits to not lose the credit for his work. Thanks.

**Comment by beni0888** [comment]
@javiereguiluz Thanks and sorry for not finish it myself

**Comment by javiereguiluz** [comment]
@beni0888 no worries! Starting things is always the most important part. Your PR "forced" us to write this cookbook ... and that's great! Thanks.

**Pull request closed without merging**